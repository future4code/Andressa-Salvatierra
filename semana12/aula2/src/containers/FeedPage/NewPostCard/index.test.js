import React from "react";
import { NewPostCard } from "./index"
import renderer from "react-test-renderer";

it("Tests if the NewPostCard (container for creating new posts) renders correctly", () => {
    const tree = renderer
        .create(<NewPostCard />)
        .toJSON()
    expect(tree).toMatchSnapshot()
})
