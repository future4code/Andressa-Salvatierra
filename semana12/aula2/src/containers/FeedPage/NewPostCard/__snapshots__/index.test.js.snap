// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests if the NewPostCard (container for creating new posts) renders correctly 1`] = `
<div
  className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
>
  <div
    className="MuiCardContent-root"
  >
    <div
      className="sc-AykKC ggeHVs"
    >
      <h5
        className="MuiTypography-root MuiTypography-h5"
      >
        Crie um novo post!
      </h5>
      <hr />
      <div
        className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
          data-shrink={false}
        >
          Título
        </label>
        <div
          className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            autoFocus={false}
            className="MuiInputBase-input MuiInput-input"
            disabled={false}
            name="title"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="Insira seu título aqui"
            required={false}
            type="title"
            value=""
          />
        </div>
      </div>
      <br />
       
      <br />
      <div
        className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
          data-shrink={false}
        >
          Conteúdo
        </label>
        <div
          className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl MuiInputBase-multiline MuiInput-multiline"
          onClick={[Function]}
        >
          <textarea
            aria-invalid={false}
            autoFocus={false}
            className="MuiInputBase-input MuiInput-input MuiInputBase-inputMultiline MuiInput-inputMultiline"
            disabled={false}
            name="text"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="Insira seu conteúdo aqui"
            required={false}
            rows="2"
            value=""
          />
        </div>
      </div>
      <br />
      <br />
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-text"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          Publicar post
        </span>
      </button>
    </div>
  </div>
</div>
`;
